### PR TITLE
Replace simulating a verifying process page in account flow

### PR DESCRIPTION
**Pull request by sanderPostma**

### CODE REVIEW

File: packages/oid4vci-demo-frontend/public/locales/development/en/translation.json [hunk 1]
```diff
@@ -80,8 +80,11 @@
   "title_personal_details": "Personal Details",
   "title_secure_your_credential": "Secure your {{credentialName}} credential",
   "title_secure_your_personal_credential": "Secure your personal credential",
+  "title_please_check_your_email": "Please check your e-mail",
+  "title_fill_5_digits": "Fill in the 5-digit code from the confirmation e-mail.",
   "paragraph_add_credential": "Let's add the credential to your wallet",
   "paragraph_scan_this_qr": "Scan this QR code using your wallet's QR scanner. When you already on your phone open your wallet by clicking on the Open in wallet button.",
+  "paragraph_confirm_email": "Confirm your Account by entering the 5 digit confirmation code from the email.",
```
Comment (BtencateSphereon) [inline packages/oid4vci-demo-frontend/public/locales/development/en/translation.json]: small thing but i think it should be: Confirm your account....

File: packages/oid4vci-demo-frontend/src/components/VerificationCodeField/index.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    }
+
+    return (
+        <div className={style.container}>
+            {values.map((value, index) => (
+                <input
+                    ref={(el) => (inputRefs.current[index] = el)}
+                    className={style.digitField}
+                    key={index}
+                    type="text"
```
Comment (BtencateSphereon) [inline packages/oid4vci-demo-frontend/src/components/VerificationCodeField/index.tsx:72]: type: number

**Review by BtencateSphereon — CHANGES_REQUESTED** [review]

File: packages/oid4vci-demo-frontend/public/locales/development/en/translation.json (hunk 1 above)
Comment (sanderPostma) [reply packages/oid4vci-demo-frontend/public/locales/development/en/translation.json]: fixed

File: packages/oid4vci-demo-frontend/src/components/VerificationCodeField/index.tsx (hunk 2 above)
Comment (sanderPostma) [reply packages/oid4vci-demo-frontend/src/components/VerificationCodeField/index.tsx:72]: Not that, that will try to cram up/down buttons in there. implemented onInput instead

Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/components/VerificationCodeField/index.tsx:72]: well, you could have used some css to hide those arrows. but this is fine

**Review by BtencateSphereon — APPROVED** [review]

**Pull request merged**